tracker/view/login.py

```diff
@@ -6,34 +6,46 @@
 from flask_login import logout_user
 from werkzeug.exceptions import Unauthorized
 
+from config import SSO_ADMINISTRATOR_GROUP
+from config import SSO_ENABLED
+from config import SSO_GUEST_GROUP
+from config import SSO_NEW_USER_DEFAULT_PASSWORD
+from config import SSO_REPORTER_GROUP
+from config import SSO_SECURITY_TEAM_GROUP
 from config import TRACKER_PASSWORD_LENGTH_MAX
 from config import TRACKER_PASSWORD_LENGTH_MIN
+from tracker import oauth
 from tracker import tracker
 from tracker.form import LoginForm
 from tracker.model.user import User
 from tracker.user import user_assign_new_token
 from tracker.user import user_invalidate
 
+from ..model.enum import UserRole
+
```

Comment:
We can think about using such pattern globally, but let's keep the current coding style (the line needs to be moved with isort afterwards) :cat: 
```suggestion
from tracker.modelenum import UserRole
```